lib/wfsc/falco_wfsc_loop.m

```diff
@@ -145,9 +157,10 @@
 
     %% SAVE THE TRAINING DATA OR RUN THE E-M Algorithm
     if mp.flagTrainModel; mp = falco_train_model(mp,ev); end
-
+    
+    %% end early? you can change the value of bEndEarly in debugger mode, but you cannot change mp.Nitr or Itr
     if flagBreak
-        break
+        break;

```

Comment:
flagBreak is unnecessary. one would need to set a breakpoint anyway.

Reply:
We added this so it could more gracefully exit out of the function and still do the wrap up steps, although I don't know how important that is.